src/pages/en/guides/migration-guide/from-gatsby.md

```diff
@@ -0,0 +1,1199 @@
+---
+title: Migrating from Gatsby
+description: Tips for migrating an existing Gatsby project to Astro
+layout: ~/layouts/CMSLayout.astro
+stub: false
+framework: Gatsby
+---
+
+Here are some tips for converting a Gatsby project to Astro. This is not a full, step-by-step walkthrough, but it will guide you through some changes you will have to make. 
+
+## Key Similarities between Gatsby and Astro
+
+Gatsby and Astro share some similarities that will help you migrate your project:
+
+- The syntax of `.astro` files is similar to JSX. Writing Astro should feel familiar.
+
+- Astro is component-based. Your markup structure will likely align closely before and after your migration.
+
+- Astro has built-in support for Markdown and an integration for using MDX files. Both Gatsby and Astro use [Remark](https://remark.js.org/) by default for Markdown manipulation and pre-processing, so you can continue to use many of your existing Remark plugins.
+
+- Astro also has [official integrations for using React components](/en/guides/integrations-guide/react/). Note that in Astro, React files **must** have a `.jsx` or `.tsx` extension.
+
+- Astro projects can also be SSG or SSR. (Support for per-page rendering strategy is planned.)
+
+- Astro has support for NPM package usage, including several for React. You may be able to keep some or all of your existing React components and dependencies.
+
+## Key Differences between Gatsby and Astro
+
+When you rebuild your Gatsby site in Astro, you will notice some important differences.
+
+### React App vs MPA 
+
+Gatsby is a React app, and uses `index.js` as your project's root. While Gatsby is able to generate an HTML page for each of the routes configured, it also re-initializes React for all of the content on-screen, including static portions.
+
+Astro is a multi-page site, and `index.astro` is your home page. It generates HTML pages for each configured route but, in contrast to Gatsby, only initializes JavaScript on the interactive elements on-screen.
+
+### Page routing
+
+You don't need to know any domain-specific knowledge to link between different pages in Astro. Instead of Gatsby's custom `<Link to="/path">` component, use the HTML standard `<a href="/path">` tag.
+
+In addition, while Gatsby primarily uses the `createPages` function to create dynamic pages, [Astro opts for page-based routing](/en/core-concepts/routing/#dynamic-routes) that better represents the individual HTML files that are output in a build.
```

Comment:
```suggestion
Gatsby uses the `createPages` function to configure creating dynamic pages, whereas [Astro opts for page-based dynamic routing](/en/core-concepts/routing/#dynamic-routes). In Astro, both static pages and pages that generate multiple pages are located within `src/pages/` and share many similarities. 
```
So, what if we focus on the fact that `getStaticPaths()` *in a page file itself* creates pages, whereas Gatsby's page creation is kind of closer to a "config/setup" type thing? Would this be true? (Not sold on this wording; wondering whether this qualifies as an important diff to hit, and whether it's meaningful/helpful to point this out to those converting.)